src/index.js

```diff
@@ -1,12 +1,45 @@
-import _ from 'lodash';
 import './style.css';
 
- function component() {
-   const element = document.createElement('div');
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-   element.classList.add('hello');
+const score = document.querySelector('.score');
+const formDiv = document.querySelector('.form-div');
 
-   return element;
- }
+score.innerHTML = `<div class="header-score">
```

Comment:
**Kindly implement these mandatory changes:**
- **Use ES6 modules to write modular JavaScript with proper ES6 syntax.**


- Usage of ` arrow functions ` instead of the ` function ` keyword.
- Usage of ` import ` and ` export ` instead of ` require `  or ` module.exports `.
- Proper usage of ` const ` and ` let ` keywords.

_This information it's resumed on your project requirements:_
![Screen Shot 2021-08-31 at 21 19 37](https://user-images.githubusercontent.com/22258274/131601082-0c969f59-9973-435f-be3d-978d48ed4d89.png)